public/content/documentation/web/component/breadcrumbs.md

```diff
@@ -0,0 +1,104 @@
+## General Notes
+
+How to test a breadcrumbs
+
+## Condensed
+
+### #a11y - Web Accessibility Acceptance Criteria
+
+1. Keyboard & screen reader actions
+   - Tab: Focus visibly moves to the link
+   - Enter: Activates the link
+
+2. Mobile screenreader gestures
+   - Swipe: Focus moves to the link
+   - Doubletap: Activates the link
+
+3. Screenreader output for all devices
+   - Name: The link names correspond to their destination page titles
+   - Role: Links identify as a links in a breadcrumb navigation landmark
+   - State: The current page link is indicated when focused
+   - Group: Is discoverable with screenreader shortcuts as a navigation landmark
+
+Full information: [https://www.magentaa11y.com/checklist-web/breadcrumbs/](https://www.magentaa11y.com/checklist-web/breadcrumbs/)
```

Comment:
Lindsay had a different url structure on the button. I did the same as you on the search input but she had me fix it. 

The button has: Full information: [https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/button](https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/button)